src/__test__/list.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+import Manipulation from '../manipulateList';
+import Storage from '../storage';
+
+describe('List', () => {
+  document.body.innerHTML = `<div class="list_container">
+  <div class="title">
+    <p>Today's To Do</p>
+    <i class="material-icons">autorenew</i>
+  </div>
+  <div class="oneMoreTask">
+    <input type="text" name="add_item" id="add_item" class="add_item" placeholder="Add to your list...">
+    <i class="material-icons">keyboard_return</i>
+  </div>
+  <ul class="tasks_list"></ul>
+  <div class="clearBtn">Clear all completed</div>
+</div>`;
+
+  global.localStorage = {
+    state: {},
+    setItem(key, item) {
+      this.state[key] = item;
+    },
+    getItem(key) {
+      return this.state[key];
+    },
+  };
+  let tasks;
+  beforeAll(() => {
+    tasks = new Manipulation('ToDoList');
+
+    tasks.addToList('Test 1');
+  });
+
+  afterAll(() => {
+    localStorage.removeItem('ToDoList');
+  });
+
```

Comment:
- [ ] It would be better if we implement a test for updating an item's 'completed' status. We are missing this test. This ensures our application works as expected

Reply:
There is a completed status in the project referred to as done and it is covered in the previous test.